### DISCUSSION TITLE
Is there any option to prohibit all the Remote applications(Eg.Teamviewer, AnyDesk, etc..) & Virtual web camera softwares(Eg.YouCam,Logitech capture, etc..) in SEB Configuration?

### DISCUSSION THREAD

**Discussion by Chris251290** [SafeExamBrowser/seb-win-refactoring#458] [Q&A]
Hi Team, 

Is there any option to prohibit all the Remote applications(Eg.Teamviewer, AnyDesk, etc..) & Virtual web camera softwares(Eg.YouCam,Logitech capture, etc..) in SEB Configuration file by giving single exe file in SEB Prohibited processes?

Because, There are multiple remote sharing applications available. If we try to prohibit all the applications in SEB, we have to give the executable file in the Prohibited processes. It is a big process. If candidate share his screen from any remote sharing application or use any web camera softwares, it will create security issues. So, Please tell us how we can prohibit all the applications by using common executable file in SEB Config file? or Is there any other ways avaiable to prohiobit?

Kindly help.

**Comment by dbuechel** [comment]
No, you'll need to manually register all the applications you specifically want to prohibit in your configuration (as explained in the documentation: https://safeexambrowser.org/windows/win_usermanual_en.html#ProhibitedProcessesSection). There already are quite a few applications in the default list, so you certainly don't need to start from scratch. And if you feel like there are more common applications which should be included there, you're most welcome to suggest them here or in a pull request.

**Reply by dbuechel** [reply]
There is an option to suppress remote sessions and screen sharing, but you'll need to use the SEB Service to take advantage of it (which we do not recommend for BYOD): https://safeexambrowser.org/windows/win_usermanual_en.html#SecurityPane.
